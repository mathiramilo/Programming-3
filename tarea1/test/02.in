leer_grafo
10 dirigido
1: 2 3 .
2: 1 3 4 .
3: 2 4 5 .
4: 3 5 6 .
5: 3 6 .
6: 5 .
0: .
orden_topologico
leer_grafo
10 dirigido
1: 2 3 .
2: 3 4 .
3: 4 5 .
4: 5 6 .
5: 6 .
6: .
0: .
orden_topologico
Fin