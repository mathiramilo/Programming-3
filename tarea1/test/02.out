1>2>Hay un ciclo 
3>4>Hay orden topologico 
5>Fin.
